src/slider.test.basics.single.ts

```diff
@@ -0,0 +1,202 @@
+import { expect, fixture, html } from '@open-wc/testing';
+import Slider from './slider.js';
+
+it('is accessible ', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label"></glide-core-slider>`,
+  );
+
+  await expect(host).to.be.accessible();
+});
+
+it('sets `value` when one is not provided to 25%', async () => {
```

Comment:
As is, it can be read two different ways.

```suggestion
it('sets `value` to 25% when not provided', async () => {
```